test/copyright/Copyright.java

```diff
@@ -9,187 +9,204 @@
 import java.util.*;
 
 public class Copyright {
```

Comment:
BTW, why not making this class `static`?

Reply:
It's an isolated tool, so I didn't bother rewriting everything.